func@0.25.rb

```diff
@@ -1,6 +1,6 @@
 require 'fileutils'
 
-class Func < Formula
+class FuncAT025 < Formula
```

Comment:
Required change to have test passing. Otherwise it's very nicely reporting failure. :)